templates/web/base/header.html

```diff
@@ -22,6 +22,7 @@
         [% INCLUDE 'header/css.html' %]
         [% INCLUDE 'common_header_tags.html' %]
         [% TRY %][% PROCESS 'header_extra.html' %][% CATCH file %][% END %]
+        <link rel="stylesheet" href="/vendor/govuk-frontend/forms.css">
```

Comment:
Could this be in header/css.html (two templates), and then all the other includes of it in various header_extra.html etc can be removed.